retriever.py

```diff
@@ -0,0 +1,271 @@
+import sys
```

Comment:
Hi pleaase add the default template

https://github.com/Open-MSS/MSS/blob/develop/new_file_template.py